dotCMS/src/main/java/com/dotmarketing/portlets/containers/business/ContainerAPIImpl.java

```diff
@@ -430,6 +435,12 @@ public Container save(Container container, List<ContainerStructure> containerStr
 			}
 		}
 
+		try {
+			String bla = null;

```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused "bla" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1481)